app/javascript/chat/chat.jsx

```diff
@@ -1409,7 +1410,35 @@ export default class Chat extends Component {
       .getElementById('jumpback_button')
       .classList.remove('chatchanneljumpback__hide');
   };
-
+  handleImageDrop = (e) => {
+    document.getElementById('messagelist').style.opacity = 1;
+    dragDrop(e, this.handleImageSuccess, this.handleImageFailure);
+  };
+  handleImageSuccess = (res) => {
+    const mLink = `![alt text](${res.links[0]})`;
```

Comment:
is there a way we can generate the alt text from the file name? It's better than "alt text" and since JS has access to the filename beore it gets uploaded, we should be able to do that, right?

https://developer.mozilla.org/en-US/docs/Web/API/File

Reply:
Can be done.